react/features/toolbox/components/AbstractToolbox.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+
+import { Component } from 'react';
+
+import {
+    AUDIO_MUTE,
+    VIDEO_MUTE,
+    createToolbarEvent,
+    sendAnalytics
+} from '../../analytics';
+import {
+    VIDEO_MUTISM_AUTHORITY,
+    setAudioMuted,
+    setVideoMuted
+} from '../../base/media';
+
+export type Props = {
+
+    /**
+     * Flag showing that audio is muted.
+     */
+    _audioMuted: boolean,
+
+    /**
+     * Flag showing whether video is muted.
+     */
+    _videoMuted: boolean,
+
+    dispatch: Function
+};
+
+/**
+ * Some doc here.
+ */
+export default class AbstractToolbox extends Component<*, *> {
+    _onToggleAudio: () => void;
```

Comment:
I almost never write flow...but from what I saw yes, it's usually terminated with a semicolon. My opinion is I don't like flow because types have not been a problem for me (maybe if I get bit by types I'll realize how much I need it) so I just go with whatever the common practice is, which is adding the semicolon.